sdk/cosmosdb/cosmos/samples-dev/typescript/Shared/handleError.ts

```diff
@@ -0,0 +1,46 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+/**
+ * @summary Demonstrates using SasTokens for granting scoped access to Cosmos resources. *Private feature*
+ */
+
+import * as dotenv from "dotenv";
+dotenv.config({ path: "../../../sample.env" });
```

Comment:
The `dotenv` path should be relative to CWD by default, so this path should not be necessary. Just `dotenv.config()`.